kinto/core/schema.py

```diff
@@ -0,0 +1,110 @@
+import six
```

Comment:
I don't see many module-level docstrings on `kinto.core` (`__init__.py` is the only exception), that's why I didn't use them, but I agree it would be nice to have this written in the code somewhere.